packages/base/src/asset-registries/Illustrations.js

```diff
@@ -13,18 +14,58 @@ const registerIllustration = (name, { dialogSvg, sceneSvg, spotSvg, set, title,
 	});
 };
 
+const registerIllustrationLoader = async (illustrationName, loader) => {
+	loaders.set(illustrationName, loader);
+};
+
+const _loadIllustrationCollectionOnce = async illustrationName => {
+	let temp;
+	if (illustrationName.startsWith("Tnt")) {
+		illustrationName = illustrationName.substring(3);
+		temp = `Tnt${illustrationName}`;
+	} else {
+		temp = illustrationName;
+	}
+	if (!loaders.has(illustrationName.replace("Tnt", ""))) {
```

Comment:
I think with this line `illustrationName = illustrationName.substring(3);`, you removed "Tnt" from `illustrationName` and `illustrationName.replace("Tnt", "")` looks like can be skipped or?